system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -58,6 +58,8 @@
 
 #define PWD_NEVER_EXPIRES (__int64)0x8000000000000000
 
+#define UNK_PERM_VALUE (SecAccessFlags)-2	//used to initialize "default" permission, which we later try to deduce
```

Comment:
Could this be a formal SecAccessFlags value? Something like SecAccess_Uninitialized?